x/mongo/driver/topology/connection.go

```diff
@@ -171,6 +176,12 @@ func (c *connection) wait() error {
 	return c.connectErr
 }
 
+func (c *connection) waitForContext() {
+	if c.contextMade != nil {
```

Comment:
Won't it be the case that this is always non-nil?

Reply:
yes, fixed